pom.xml

```diff
@@ -66,9 +66,9 @@
             <!-- The test JVMs are short-running. By disabling certain
             expensive JIT optimizations we actually speed up most tests. -->
             -XX:TieredStopAtLevel=1
-            <!-- We cap memory usage. This is especially relevant on Travis CI,
-            but locally this should also be more than enough. -->
-            -Xmx512m
+            <!-- We cap memory usage. This is especially relevant on CI, but
```

Comment:
This change is also based on our internal build file.

Reply:
Hmm, we use a different text in `error-prone-support`, with the latter text seemingly of more recent vintage.